src/main/java/picard/illumina/IlluminaBasecallsToFastq.java

```diff
@@ -104,14 +99,8 @@
 
     // The following attributes define the command-line arguments
 
-    @Argument(doc = "The basecalls directory. ", shortName = "B")
-    public File BASECALLS_DIR;
-
     @Argument(doc = "The barcodes directory with _barcode.txt files (generated by ExtractIlluminaBarcodes). If not set, use BASECALLS_DIR. ", shortName = "BCD", optional = true)
-    public File BARCODES_DIR;
-
-    @Argument(doc = "Lane number. ", shortName = StandardOptionDefinitions.LANE_SHORT_NAME)
-    public Integer LANE;
+    public File BARCODES_DIR = BASECALLS_DIR;
```

Comment:
adding ` = BASECALLS_DIR` here won't have any effect because at contruction time `BASECALLS_DIR` will be null also.